modules/apps/frontend-taglib/frontend-taglib-clay/src/main/resources/META-INF/resources/data_set_display/management_bar/components/MainSearch.js

```diff
@@ -36,55 +36,44 @@ function MainSearch() {
 	}
 
 	return (
-		<div className="d-inline">
-			<div className="input-group">
-				<div className="input-group-item">
-					<div className="main-input-wrapper">
-						<input
-							className="form-control input-group-inset input-group-inset-after main-input-search"
-							onChange={(event) =>
-								updateInputValue(event.target.value)
-							}
-							onKeyDown={handleKeyDown}
-							placeholder={Liferay.Language.get('search')}
-							type="text"
-							value={inputValue}
-						/>
+		<ClayInput.Group>
+			<ClayInput.GroupItem>
+				<ClayInput
+					className="input-group-inset input-group-inset-after"
+					onChange={(event) => updateInputValue(event.target.value)}
+					onKeyDown={handleKeyDown}
+					placeholder={Liferay.Language.get('search')}
+					value={inputValue}
+				/>
+				<ClayInput.GroupInsetItem after tag="div">
+					<ClayButtonWithIcon
```

Comment:
Is it possible to not have `btn-monospaced` output? If not I should update in Clay CSS to make spacing smaller.

Reply:
Yes, I can replace it just using the low-level instead of the high-level component

Reply:
Currently, we can remove the monospaced, just by setting `monospaced={false}`, this has been [added recently](https://github.com/liferay/clay/commit/9c2e51fecf81c87f8f6af4818057b04a41ea8517).